xml/System.Windows.Forms/FolderBrowserDialog.xml

```diff
@@ -320,9 +320,9 @@
         <Parameter Name="hWndOwner" Type="System.IntPtr" />
       </Parameters>
       <Docs>
-        <param name="hWndOwner">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="hWndOwner">A handle to the window that owns the folder browser dialog.</param>
+        <summary>Runs a folder browser dialog box.</summary>
```

Comment:
- "Runs" seems like a peculiar word to describe a dialog box. Opens would be better. ("Runs" is used for the overloads of ShowDialog as well.)
- RunDialog is showing as inherited from CommonDialog, not overriding CommonDialog.RunDialog. That's true of all .NET Framework versions, as well as .NET Core.

Reply:
I was really struggling how to make this one better last night. 
This is how we describe this throughout our reference? https://docs.microsoft.com/en-us/dotnet/api/?term=RunDialog

Reply:
Also, I wouldn't worry too much about this one. This is one of those protected APIs of sealed classes so they don't seem to really show up since they're "private". But if you have any suggestions, let me know @rpetrusha.

Reply:
Changing to Displays and I'll open a new PR to make others consistent as well.